### PR TITLE
pack: generate per-application systemd service 

### PR DESCRIPTION
**Pull request by psergee**
General systemd-unit file is removed.
`tt pack` generates service systemd unit for each packed application.
Re-factor `tt pack` to use `running` package to get applications and
instances information required to generate unit files.

### CODE REVIEW

**Review by oleg-jukovec — APPROVED** [review]

File: cli/pack/opts.go [hunk 1]
```diff
@@ -15,9 +21,45 @@ const (
 	Docker = "docker"
 )
 
+func initAppsInfo(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx) error {
```
Comment (better0fdead) [inline cli/pack/opts.go:24]: It would be nice to add a comment. Up to you

**Review by better0fdead — APPROVED** [review]
Hi, thx for patch

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+
+- `tt pack` generates a separate systemd unit for each packed application.
+Common (all instances) unit is removed.
+
+### Fixed
+
+- `tt pack` and `tt build` fail in verbose mode with "invalid argument"
+error
```
[the remark this answers is not in the document]
Comment (psergee) [reply CHANGELOG.md]: Fixed.

File: cli/cmd/pack.go [hunk 3]
```diff
@@ -101,7 +101,7 @@ func internalPackModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 		return errNoConfig
 	}
 
-	err := pack.FillCtx(cmdCtx, packCtx, args)
+	err := pack.FillCtx(cliOpts, cmdCtx, packCtx, args)
```
[the remark this answers is not in the document]
Comment (psergee) [reply cli/cmd/pack.go]: Fixed.

File: cli/pack/opts.go (hunk 1 above)
Comment (psergee) [reply cli/pack/opts.go:25]: Comment is added.

**Pull request merged**